_LowLevelCode/cpp/serialiser/c_deserialise.cpp

```diff
@@ -342,6 +343,13 @@ mxArray* deserialise(uint8_t* data, size_t& memPtr, size_t size, bool recursed)
 
 /* MATLAB entry point c_deserialise */
 void mexFunction(int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[] ) {
+    
+  //--------->  RETURN MEX-file version if requested;
+  if (nrhs == 0 && (nlhs == 0 || nlhs == 1)) {
+        plhs[0] = mxCreateString(Herbert::VERSION);
```

Comment:
Is this expected behaviour? If you ask me to serialise nothing, I would expect to receive nothing (or a serialised nothing).

Reply:
This is implemented in all other mex functions and used to check if mex files are compiled and links are satisfied on initialization. 
All Horace/Herbert mex files have this option. See `check_horace_mex`; `check_herbert_mex` Matlab routines. 
Extremely useful for peoples who may install Horace without cmake.